edifact-edi/2.1/modules/ROOT/pages/index.adoc

```diff
@@ -276,6 +276,7 @@ Settings for writing messages:
 * Enforce valid characters for UNOA and UNOB syntax levels.
 * Request acknowledgments for send interchanges using Acknowledgment Request field (UNB.9).
 * Set a test indicator digit to be used on send interchanges (UNB.11).
+* Whether to use supplied values for control segment information such as control numbers.
```

Comment:
```suggestion
* Whether to use supplied values for control-segment information such as control numbers.
```

Reply:
Is the information ONLY control numbers or mostly control numbers: can we say "Whether to use supplied values for control numbers."?

Reply:
I think not, but confirming.

Reply:
It's a not.